contributors.yml

```diff
@@ -215,3 +215,6 @@
 - yionr
 - yuleicul
 - zheng-chuang
+- nnhjs
+- gloomweaver
+- Ivanrenes
```

Comment:
It seems like something went wrong here 🤔

Reply:
Yeah, fixed